README.Rmd

```diff
@@ -237,7 +249,37 @@ project by clicking the `.Rproj` file inside the project folder.
 
 Then, open up the README.md file in your project and follow the instructions
 there. When you are ready to create the dataset necessary for your project,
-open up the `data-raw/download-data.R` script and follow the instructions there.
+open up the `data-raw/download-data.R` script and follow the instructions there
+to create your dataset.
+
+Remember to push your code to your GitHub. Otherwise all your work on the RAP
+will be lost next time you login. Also remember to terminate your session when
+you are done. 
+
+### Next time you go into UKB RAP
+
+Next time you open up the RAP you will:
```

Comment:
```suggestion
Next time you open up the RAP you will need to follow these steps below.
```

I am actually updating this section to make it more streamlined (from suggestion by Jonas), so it will be modified shortly :relaxed: